### PR TITLE
Project proposal: NGE

### PR DESCRIPTION
**Pull request by Keller-Peter**
Questionnaire for Netzgrafik-Editor (NGE)

### CODE REVIEW

File: project_proposals/NGE.md [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+
+
+## What is the project's leadership team and decision-making process?
+tbd
+
+## How is it decided if and when a pull request is merged?
+Pull requests are reviewed and merged by the maintainers.
+
+## How can someone become a committer or a maintainer to/of the project?
+Contributer: via Issues; pull requests - Maintainer tbd
```
Comment (SergeCroise) [inline project_proposals/NGE.md:141]:
```suggestion
Contributor: via Issues; pull requests - Maintainer tbd
```

File: project_proposals/NGE.md [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+  flexibility and features to effectively visualize and plan their transportation systems,
+  leading to improved efficiency and better services.
+ 
+- **Educations:** The editor's capabilities can be utilized in educational institutions to demonstrate
+  how to create integrated timetables and what properties need to be considered during the planning
+  process in order to enable efficient scheduling
+
+
+## Why would this project be a good candidate for inclusion in the OpenRail Association?
+- With its clear focus on the rapid recording of timetable variants on a macroscopic level, Netzgrafik-Editor is suitable as a tool that can be combined with other OSS tools in the area of timetable planning and simulation (e.g. OSRD; MATsim).
```
Comment (SergeCroise) [inline project_proposals/NGE.md:166]:
```suggestion
- With its clear focus on the rapid recording of timetable variants on a macroscopic level, Netzgrafik-Editor is suitable as a tool that can be combined with other OSS tools in the area of timetable planning and simulation (e.g., OSRD; MATsim).
```

File: project_proposals/NGE.md [hunk 3]
```diff
@@ -0,0 +1,190 @@
...
+## Are there competing products or projects? If there are please explain how the proposed projects differentiates.
+- With this scope on the macroscopic not known.
+
+
+## What standards does the project implement or rely on? How are they related to other existing standards?
+None
+
+
+## What is the tech stack of the project? Name the major programming languages and frameworks which are used.
+[angular.sbb.ch](https://angular.app.sbb.ch/)
```
Comment (cornelius) [inline project_proposals/NGE.md:179]: It would be good to mention the tech stack of the backend as well (Spring Boot with PostgreSQL backend)

**Review by cornelius — APPROVED** [review]
Proposal looks good now. As we discussed in the TC meeting, the TC suggests to accept the project.

**Comment by cornelius** [comment]
Board has approved. Merging this now.

**Pull request merged**